go.mod

```diff
@@ -13,3 +13,5 @@ require (
 	github.com/zclconf/go-cty v1.8.3
 	github.com/zclconf/go-cty-debug v0.0.0-20191215020915-b22d67c1ba0b
 )
+
+replace github.com/hashicorp/hcl-lang v0.0.0-20210603100913-81056fd9a6e0 => github.com/aztfmod/hcl-lang v0.0.0-20210607111743-09c4e3d54cdd
```

Comment:
hcl-lang should be updated when https://github.com/hashicorp/hcl-lang/pull/47 is merged